lib/util/common.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @fileoverview Utility classes and functions for common tasks.
+ * @author Johnny Zabala
+ */
```

Comment:
i'd prefer to just stick this in its own file; there's not a lot of value from having a big grab bag file.

Reply:
Done!